website/docs/docs/use-dbt-semantic-layer/exports.md

```diff
@@ -0,0 +1,122 @@
+
+## Define exports
+
+Exports are essentially saved queries that you can schedule and execute using [dbt's job scheduler](/docs/deploy/job-scheduler) capabilities. Exports include the following materialization types: CAN A USER ALSO USE API OR CI JOBS? WHAT IF THEY USE ANOTHER ORCHESTRATOR?
+
+- `table` (available now) 
+- and `window_table`, `incremental_table`, and `file` (coming soon)
+
+You can define Exports in a YAML format in the same file as saved queries:
+
+```yaml
+saved_queries:
+  - name: my_query
+    description: null
+    query_params:
+      metrics:
+        - null
+      group_bys:
+        - TimeDimension()
+        - ...
+      where:
+        - null
+    exports:
+      - name: my_query
+        config:
+          export_as: table # options: table, view
+          schema: my_schema # [optional - DEFAULT to deployment schema]
+          alias: some_table_name # [optional - DEFAULT to export name]
+```
+
+## Parameters
+
+By default all exports are run for a saved query. You can select a specific export with the `select` flag or you can specify a new export using the `export-as` flag. The Job scheduler runs the equivalent of` dbt sl export --saved-query sq_name`.
+
+| Parameters | Type    | Required | Description    |
+| ------- | --------- | ---------- | ---------------- |
+| `saved-query` | String    | Required     | A name of a saved query that could be used.    |
+| `select` | List or String   | Optional    | Names of exports to be selected from the saved query. |
+| `exclude` | String  | Optional    | Not the names of exports to be selected from the saved query. |
+| `export-as` | String  | Optional    | Type of export to create from the export_as types available in the config. |
+| `schema` | String  | Optional    | Schema to use for creating the table or view. |
+| `alias` | String  | Optional    | Table alias to use to write the table or view. |
+
+## Schedule Exports
+
+Exports in dbt are effectively integrated with your data models and metrics, allowing you to automate and materialize saved queries using the dbt Cloud job scheduler. These exports, along with saved queries, are treated as standard dbt models and are executed as part of the dbt <Term id="dag" />, following model builds where dependencies exist.
+
+To execute exports using the job scheduler:
+
+- Create a [deploy job](/docs/deploy/deploy-jobs) in dbt Cloud. DO USERS NEED TO HAVE A SUCCESSFUL JOB RUN BEFORE THEY CAN SCHEDULE EXPORTS? WHAT IF THEY DON'T HAVE A SUCCESSFUL JOB RUN?
+- Use the built-in `dbt build` command to execute exports and saved queries. Note that although you can selectively run specific saved queries, this level of selection isn't available for individual exports. However, you can always create a different saved query if you want to separate exports. WHAT'S A SAMPLE QUERY? IS IT `export-as export_name` OR `dbt sl export --saved-query sq_name`
```

Comment:
DO USERS NEED TO HAVE A SUCCESSFUL JOB RUN BEFORE THEY CAN SCHEDULE EXPORTS? WHAT IF THEY DON'T HAVE A SUCCESSFUL JOB RUN?